package.json

```diff
@@ -4,10 +4,9 @@
   "description": "initials for names",
   "main": "initials.js",
   "scripts": {
-    "jshint": "jshint -c .jshintrc *.js test/*.js",
-    "test": "npm run jshint && testmate test/*.js",
-    "test:node": "testmate --client node test/*.js",
-    "test:browser": "testmate --client selenium:firefox test/*.js",
+    "test": "standard initials.js test/initials-test.js && npm run -s test:node | tap-spec",
```

Comment:
can you move "standard" into "pretest"? And if you want to ignore files / folders from being linted by standard, check https://www.npmjs.com/package/standard#how-do-i-ignore-files